src/bootstrap/config.rs

```diff
@@ -671,6 +650,28 @@ impl Config {
                 config.llvm_link_shared = true;
             }
 
+            set(&mut config.ninja_in_file, llvm.ninja);
+            llvm_assertions = llvm.assertions;
+            llvm_skip_rebuild = llvm_skip_rebuild.or(llvm.skip_rebuild);
+            set(&mut config.llvm_optimize, llvm.optimize);
+            set(&mut config.llvm_thin_lto, llvm.thin_lto);
+            set(&mut config.llvm_release_debuginfo, llvm.release_debuginfo);
+            set(&mut config.llvm_version_check, llvm.version_check);
+            set(&mut config.llvm_static_stdcpp, llvm.static_libstdcpp);
+            config.llvm_targets = llvm.targets;
+            config.llvm_experimental_targets = llvm.experimental_targets;
+            config.llvm_link_jobs = llvm.link_jobs;
+            config.llvm_version_suffix = llvm.version_suffix;
+            config.llvm_clang_cl = llvm.clang_cl;
+
+            config.llvm_cflags = llvm.cflags;
+            config.llvm_cxxflags = llvm.cxxflags;
+            config.llvm_ldflags = llvm.ldflags;
+            set(&mut config.llvm_use_libcxx, llvm.use_libcxx);
+            config.llvm_use_linker = llvm.use_linker;
+            config.llvm_allow_old_toolchain = llvm.allow_old_toolchain;
+            config.llvm_from_ci = llvm.download_ci_llvm.unwrap_or(false);
```

Comment:
Why was this moved after the checking? I preferred the old order, and in any case, this won't work because llvm_from_ci will always be false in the if above.

Reply:
Hmm, I moved it because otherwise there was a borrow error that `llvm` had been moved. I could either move `             config.llvm_from_ci = llvm.download_ci_llvm.unwrap_or(false);` above or change the checking to use `config.llvm` instead.

Reply:
Ok, I changed it to use `config.llvm_*` instead.